components/camera/videosource/logging/logger.go

```diff
@@ -1,304 +1,2 @@
-// Package logging is a thread-safe way to log video device information to a file. On startup, this package creates a
-// unique filename and uses that filename throughout the lifetime of the program to log information such as which video
-// devices are V4L2 compatible and the current operating system.
+// Package logging is to be removed
```

Comment:
Why not remove in this PR again?

Reply:
`TestShutdown` started doing weird things when I removed the packages themselves, going to do that in a separate commit when I figure out why.